src/rust/engine/Cargo.lock

```diff
@@ -3325,6 +3318,15 @@ dependencies = [
  "winapi 0.3.9",
 ]
 
+[[package]]
+name = "time"
+version = "0.3.5"
```

Comment:
Seems like `time` v0.1.44 is still in the dependency graph?

Reply:
It is, but - afaict - that's not changeable without fixing its dependee chain which includes chrono which has no further upgrade available.